modules/cnv-networking-wizard-fields-web.adoc

```diff
@@ -16,7 +16,7 @@
 |Name for the Network Interface Card.
 
 |Model
-|Driver for the Network Interface Card or model for the Network Interface Card.
+|Indicates the model of the Network Interface Card. Supported values are e1000, e1000e, ne2k_pci, pcnet, rtl8139, and virtio.
```

Comment:
If these are literal values for the Model then each one should be in backticks - 
> `e10001`, e1000e`, etc

Reply:
Unless they're options in a drop-down menu, in which case the markup should be *